### PR TITLE
New post: Complete guide to token auth in aspnetcore

### PR DESCRIPTION
**Pull request by nbarbettini**
New post scheduled for 2018-03-27. Updates one of my post popular posts from the old Stormpath blog.

Doesn't need any code QA, just a read-through sanity check. 😄

### CODE REVIEW

**Comment by nbarbettini** [comment]
@lindsaybrunner-okta One of the h2s ended up wrapping in a nonpretty way:

image

Want to shorten it?

**Comment by lindsaybrunner-okta** [comment]
can it be generate instead of generating? I’m surprised i let generating thru

--
Lindsay Brunner
Content Marketing Manager, Developer + Security
330-704-4005
lindsay.brunner@okta.com
[ta • Connect everything]<https://www.okta.com/>

On Mar 20, 2018, 10:06 PM -0700, Nate Barbettini <notifications@github.com>, wrote:

@lindsaybrunner-okta<https://github.com/lindsaybrunner-okta> One of the h2s ended up wrapping in a nonpretty way:

[image]<https://user-images.githubusercontent.com/7525482/37695189-a28c1106-2c8a-11e8-944d-6c65298c271a.png>

Want to shorten it?

File: _source/_posts/2018-03-27-token-authentication-aspnetcore-complete-guide.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+IdentityServer is a good choice when you want to roll your own full-fledged OpenID Connect authorization server that can handle complex use cases like federation and single sign-on. Depending on your use case, configuring IdentityServer4 can be a little complicated. Fortunately, the [official documentation](http://docs.identityserver.io/en/release/) covers many common scenarios.
+
+## Token Authentication Can Be Complex!
+I hope this article helps it feel a little less confusing. The ASP.NET Core team has done a great job of making it easy to add token authentication to your ASP.NET Core API, and options like OpenIddict and Okta make it easy to spin up an authorization server that generates tokens for your clients.
+
+Here are some more resources if you want to keep learning:
+* [How to Secure Your .NET Web API with Token Authentication](https://developer.okta.com/blog/2018/02/01/secure-aspnetcore-webapi-token-auth) here on the Okta blog
```
Comment (mraible) [inline _source/_posts/2018-03-27-token-authentication-aspnetcore-complete-guide.md]: developer.okta.com/blog should be relative links > /blog

**Comment by mraible** [comment]
Looks good, but I would recommend changing `https://developer.okta.com/blog` to `/blog`

File: _source/_posts/2018-03-27-token-authentication-aspnetcore-complete-guide.md (hunk 1 above)
Comment (nbarbettini) [reply _source/_posts/2018-03-27-token-authentication-aspnetcore-complete-guide.md]: Thanks for this catch!

**Comment by nbarbettini** [comment]
Merging ahead of the publish date so it's in the queue for automatic publishing. :)

**Pull request merged**